Yoast/Yoast-SEO-for-TYPO3
# [BUGFIX] Fixed the display of the title progressbar in CMS11 with a (temporary?) dirty fix

## Summary

This PR can be summarized in the following changelog entry:

* The title bar wasn't shown in CMS11 because of the new css grid in the backend
* Sometimes(?) the pageTitleElement was not (yet) available while running the placeholder code which resulted in a javascript error with `addEventListener`

## Relevant technical choices:

* Because of the differences between CMS9, CMS10 and CMS11 a new function has been added which checks the parent elements for grid displays
* Added a check if the `pageTitleElement` is available before adding the event listener

## Test instructions

This PR can be tested by following these steps:

* Pull branch and check in CMS9, CMS10 and CMS11 if the title (and description) progress bar shows correctly

## Quality assurance


Fixes #425 #506
